ssl/record/rec_layer_s3.c

```diff
@@ -54,8 +50,10 @@ void RECORD_LAYER_clear(RECORD_LAYER *rl)
 
 void RECORD_LAYER_release(RECORD_LAYER *rl)
 {
-    if (rl->numwpipes > 0)
-        ssl3_release_write_buffer(rl->s);
+    /*
+     * TODO(RECLAYER): Need a way to release the write buffers in the record
+     * layer on demand
+     */
```

Comment:
I did notice the change from a `RECORD_LAYER_get_wbuf()` call to direct memory allocation in `ssl/d1_lib.c`, I assume that this comment is kinda sorta connected to that change...